pkg/server/handler/v1alpha1/integration.go

```diff
@@ -13,11 +13,12 @@ import (
 	"k8s.io/client-go/util/retry"
 
 	"github.com/caicloud/cyclone/pkg/apis/cyclone/v1alpha1"
+	"github.com/caicloud/cyclone/pkg/common"
 	"github.com/caicloud/cyclone/pkg/meta"
 	api "github.com/caicloud/cyclone/pkg/server/apis/v1alpha1"
 	"github.com/caicloud/cyclone/pkg/server/biz/scm"
 	"github.com/caicloud/cyclone/pkg/server/biz/usage"
-	"github.com/caicloud/cyclone/pkg/server/common"
+	svrcommon "github.com/caicloud/cyclone/pkg/server/common"
```

Comment:
What about renaming or refacting `pkg/common`, `pkg/server/common` and `pkg/workflow/common`?  It is very hard to import them now.
Of cause, this can be done in another PR.